src/inspector_agent.cc

```diff
@@ -921,6 +922,73 @@ void Agent::FatalException(v8::Local<v8::Value> error,
   impl->FatalException(error, message);
 }
 
+bool InspectorOptions::ParseOption(const char* option) {
+  bool consume = false;
+  const char* port = nullptr;
+
+  if (!strncmp(option, "--debug=", sizeof("--debug=") - 1)) {
```

Comment:
I would prefer `strlen` over `sizeof(..) - 1`. Is `sizeof` better than `strlen` here?